hw04Logging/src/main/java/Demo.java

```diff
@@ -0,0 +1,61 @@
+
+import java.lang.reflect.InvocationHandler;
+import java.lang.reflect.Method;
+import java.lang.reflect.Parameter;
+import java.lang.reflect.Proxy;
+import java.util.HashSet;
+import java.util.Set;
+
+public class Demo {
+
+
+    public static void main(String[] args) {
+        createTest().calculation(8, 10);
+        createTest().calculation(9, 11);
+        createTest().print("string");
+    }
+
+    static class MyProxy implements InvocationHandler{
+        private final Object target;
+        private Set<Method> set = new HashSet<>();
```

Comment:
```suggestion
        private Set<Method> methods = new HashSet<>();
```
Лучше назвать не ```set``` по структуре данных, в которой храниться инфа, а по тому, что там храниться.